src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ConstraintGammas.hpp

```diff
@@ -95,16 +107,21 @@ template <size_t SpatialDim, typename Frame>
 struct ConstraintGamma2Compute : ConstraintGamma2, db::ComputeTag {
   using argument_tags =
       tmpl::list<DampingFunctionGamma2<SpatialDim, Frame>,
-                 domain::Tags::Coordinates<SpatialDim, Frame>>;
+                 domain::Tags::Coordinates<SpatialDim, Frame>, ::Tags::Time,
+                 ::domain::Tags::FunctionsOfTime>;
   using return_type = Scalar<DataVector>;
 
   static constexpr void function(
       const gsl::not_null<Scalar<DataVector>*> gamma,
       const ::GeneralizedHarmonic::ConstraintDamping::DampingFunction<
           SpatialDim, Frame>& damping_function,
-      const tnsr::I<DataVector, SpatialDim, Frame>& coords) noexcept {
+      const tnsr::I<DataVector, SpatialDim, Frame>& coords, const double time,
+      const std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+          functions_of_time) noexcept {
     destructive_resize_components(gamma, get<0>(coords).size());
-    get(*gamma) = get(damping_function(coords));
+    get(*gamma) = get(damping_function(coords, time, functions_of_time));
```

Comment:
I think we discussed that at some point we want to move the `damping_function`s to returning by `gsl::not_null` right? Not a change request, just a question :)

Reply:
I don't recall discussing this, but that could be my fault. I'm happy to update them to return by not-null in a future PR, but until performance of the damping functions is measured to be a problem, I'd rather leave this for later or for a student project

Reply:
Thanks for taking such a quick look, by the way, @nilsdeppe ! much appreciated :)

Reply:
If my memory is correct, you showed a proposed interface for the base class of the damping functions in one of the calls and I asked about switching to `not_null`s. The agreement as I remember was "After X" where "X" was adding some damping function. I don't remember if it was the first damping function or later ones. In either case, we have `not_null` of almost all functions because we know doing a lot of memory allocations is bad (maybe some individual ones aren't, but there are some that we cannot avoid). This is partly why all compute tags are now using `not_null`. I also don't think these are the types of "conform to our existing practices" should be pushed to other people, e.g. the person doing the profiling.

Reply:
That must be what happened, and I couldn't remember. Anyway, I agree with what you're saying. After I get this and the triple gaussian in, I'm happy to do one last PR in this series to change the interface to `not_null`. Does that sound like an OK plan?

Reply:
Yep, that sounds great. Thank you! :D